etc/profile-a-l/electron-cash.profile

```diff
@@ -0,0 +1,54 @@
+# Firejail profile for electron-cash
+# Description: Lightweight Bitcoin Cash wallet
+# This file is overwritten after every install/update
+# Persistent local customizations
+include electron-cash.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.electron-cash
```

Comment:
Firejail's logic assumes a `noblacklist foo` is warranted when a `blacklist foo` exists somewhere in our other included files. So you'll need to add `blacklist ${HOME}/.electron-cash` to `disable-programs.inc` to protect that path from being accessible in other application sandboxes.

Reply:
That makes sense, I added the `blacklist` to `disable-programs.inc`.